src/apps/utils.ts

```diff
@@ -145,3 +145,17 @@ export const getAppInProgressName = (
   id: string,
   collection?: AppInstallationFragment[],
 ) => collection?.find(app => app.id === id)?.appName || id;
+
+export const groupIntoPairs = <T>(list: T[]): T[][] => {
```

Comment:
**suggestion**: what about using lodash [chuck](https://lodash.com/docs/#chunk) here?

As we already have lodash in the dashboard maybe it will be worth using chunk instead custom implementation?